server/src/main/java/com/objectcomputing/checkins/services/memberprofile/currentuser/CurrentUserDTO.java

```diff
@@ -0,0 +1,133 @@
+package com.objectcomputing.checkins.services.memberprofile.currentuser;
+
+import io.micronaut.core.annotation.Introspected;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.annotation.Nullable;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Past;
+import java.time.LocalDate;
+import java.util.UUID;
+
+@Introspected
+public class CurrentUserDTO {
+
+    @NotNull
+    @Schema(description = "id of the member profile this entry is associated with", required = true)
+    private UUID id;
+
+    @NotNull
+    @Schema(description = "full name of the employee", required = true)
+    private String name;
+
+    @Nullable
+    @Schema(description = "employee's role at the company", required = true)
+    private String role ;
+
+    @Nullable
+    @Schema(description = "employee's professional development lead")
+    private UUID pdlId;
+
+    @Nullable
+    @Schema(description = "where the employee is geographically located", required = true)
```

Comment:
Another required/nullable mismatch here.

Reply:
I've updated this in my latest commit.